bluesky/run_engine.py

```diff
@@ -1688,15 +1690,22 @@ def _wait(self, msg):
 
         Expected message object is:
 
-            Msg('wait', group=<GROUP>)
+            Msg('wait', group=<GROUP>, *, progress=False)
 
         where ``<GROUP>`` is any hashable key.
         """
         if msg.args:
             group, = msg.args
         else:
             group = msg.kwargs['group']
-        futs = list(self._groups.pop(group, []))
+        statuses, futs = list(zip(*self._groups.pop(group)))
```

Comment:
Yes; see [this comment](https://github.com/NSLS-II/bluesky/pull/525#issuecomment-236270694) above.